### ISSUE TITLE
1.12 battlesuit

### ISSUE THREAD

**Issue by Spenny519** [FantasyTeddy/QuantumFlux#9]
Any chance that we will see the battlesuit return to quantum flux in 1.12?

**Comment by FantasyTeddy** [comment]
Yes, this chance is absolutely real. I don't know when, but I plan to add the battlesuit again.
